oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/util/TermQueryBuilderFactory.java

```diff
@@ -128,20 +115,17 @@ public static TermQueryBuilder newNullPropQuery(String propName) {
         } else if (pr.first != null && pr.last != null) {
             return newRangeQuery(propertyName, first, last,
                     pr.firstIncluding, pr.lastIncluding);
-        } else if (pr.first != null && pr.last == null) {
+        } else if (pr.first != null) {
```

Comment:
This change seems to be unrelated to the oak issue. I see that technically, it's not needed.

Reply:
Correct. This code was inherited from the initial Elastic implementation. This check is redundant and can be safely removed to tidy up the code.